src/masonite/middleware/route/VerifyCsrfToken.py

```diff
@@ -65,7 +65,7 @@ def in_exempt(self, request):
 
     def get_token(self, request):
         return (
-            request.header("X-CSRF-TOKEN")
-            or request.header("X-XSRF-TOKEN")
+            request.header("X-CSRF-TOKEN").value
```

Comment:
Here I added a fix because headers are returned not the value... (we could also edit the request.header() method to return directly the value, I don't know what's best, maybe sometimes we would like to interact with the header object directly ?)

Reply:
The problem now is that when a header is not request.header("MYHEADER") will return "" and `.value` won't work, what would you do here ?

Reply:
I finally updated it the same way as for cookie so we don't have to call value

Reply:
> The problem now is that when a header is not request.header("MYHEADER") will return "" and .value won't work, what would you do here ?

Can the `optional()` helper be used here? to case of don't throw a invalid property of None or empty value..

Reply:
It could be actually it's the design that was not good I guess, using optional would feel like a patch.
I have finally use same method as for cookies (cf above comment)